flytekit/clis/sdk_in_container/utils.py

```diff
@@ -113,7 +113,6 @@ def pretty_print_traceback(e: Exception, verbosity: int = 1):
     Print the traceback in a nice formatted way if verbose is set to True.
     """
     console = Console()
-    tb = e.__cause__.__traceback__ if e.__cause__ else e.__traceback__
```

Comment:
I think showing all the stack trace is better, not just the e.__cause__.

Reply:
Can you add a few examples of the before and after this change?

Reply:
before:
<img width="1220" alt="Screenshot 2024-08-17 at 1 51 45 AM" src="https://github.com/user-attachments/assets/baec6ea4-355b-4248-906a-686dc587cb9e">

after:
<img width="1220" alt="Screenshot 2024-08-17 at 1 52 17 AM" src="https://github.com/user-attachments/assets/ee434686-2d10-4410-9d02-d35590f9f088">

Reply:
```
pyflyte -vv register flyte-example/getting_started.py
```